lyft_dataset_sdk/lyftdataset.py

```diff
@@ -1080,14 +1079,15 @@ def render_annotation(
         lidar = sample_record["data"]["LIDAR_TOP"]
         data_path, boxes, camera_intrinsic = self.lyftd.get_sample_data(lidar, selected_anntokens=[ann_token])
         LidarPointCloud.from_file(data_path).render_height(axes[0], view=view)
-        for box in boxes:
-            c = np.array(self.get_color(box.name)) / 255.0
-            box.render(axes[0], view=view, colors=(c, c, c))
-            corners = view_points(boxes[0].corners(), view, False)[:2, :]
-            axes[0].set_xlim([np.min(corners[0, :]) - margin, np.max(corners[0, :]) + margin])
-            axes[0].set_ylim([np.min(corners[1, :]) - margin, np.max(corners[1, :]) + margin])
-            axes[0].axis("off")
-            axes[0].set_aspect("equal")
+        box = boxes[0]
```

Comment:
IMO it would be great to add an assert or write in a way that will fail in case there are more than one boxes, e.g. 
```suggestion
        box, = boxes
```

and same with removing the loop below.